src/main/java/io/netty/incubator/codec/http3/Http3PushStreamInitializer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.incubator.codec.http3;
+
+import io.netty.channel.ChannelInitializer;
+import io.netty.incubator.codec.quic.QuicStreamChannel;
+
+abstract class Http3PushStreamInitializer extends ChannelInitializer<QuicStreamChannel> {
```

Comment:
Should this class be public as `Http3PushStreamServerInitializer` is public ? I think it should and we should just add a package-private constructor.

Reply:
I think it is OK since the `initPushStream` method is `protected` and hence available to any child class even if outside the package.

Reply:
As far as I know there were some issues when doing something like this in the past when using other languages that run on the JVM. So just make it public imho

Reply:
kk lemme remove this class as @trustin suggested below.